File: apps/problems/sample-1/boilerplate-full/function.cpp
#include <iostream>
#include <fstream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>

##USER_CODE_HERE##

int main() {
  std::ifstream file("/dev/problems/sample/tests/inputs/##INPUT_FILE_INDEX##.txt");
  std::vector<std::string> lines;
  std::string line;
  while (std::getline(file, line)) lines.push_back(line);

  file.close();
  unknown arr;
  std::istringstream(lines[0]) >> arr;
  int result = example(arr);
  std::cout << result << std::endl;
  return 0;
}

File: apps/problems/sample-1/boilerplate-full/function.rs
use std::fs::read_to_string;
use std::io::{self};
use std::str::Lines;

##USER_CODE_HERE##

fn main() -> io::Result<()> {
  let input = read_to_string("/dev/problems/sample/tests/inputs/##INPUT_FILE_INDEX##.txt")?;
  let mut lines = input.lines();
  let arr: unknown = lines.next().unwrap().parse().unwrap();
  let result = example(arr);
  println!("{}", result);
  Ok(())
}
